internal/providers/terraform/terragrunt_hcl_provider.go

```diff
@@ -561,7 +567,10 @@ func (p *TerragruntHCLProvider) runTerragrunt(opts *tgoptions.TerragruntOptions)
 			if mod.Module != nil {
 				path = mod.Module.RootPath
 			}
-
+			info.AddWarning(schema.ProjectDiag{
+				Code:    schema.DiagModuleEvaluationFailure,
```

Comment:
```suggestion
				Code:    schema.DiagModuleEvaluationFailure,
```

Could we change this to something Terragrunt related e.g. `schema.DiagTerragruntModuleEvaluationFailure` so we could distinguish in the UI/analysis later